go.mod

```
@@ -82,6 +84,9 @@ require (
 )
 
 require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.1.0 // indirect
+	github.com/Azure/azure-sdk-for-go/sdk/internal v1.0.0 // indirect
+	github.com/AzureAD/microsoft-authentication-library-for-go v0.5.1 // indirect
```

It's better to manually update this version to a newer version instead of using a `replace` rule (because `replace` rules are not inherited by consumers of this module).

When doing so, it's good to add a comment, so that future visitors of this file know that the version is changed, and why it's done;

```suggestion
	github.com/AzureAD/microsoft-authentication-library-for-go v0.6.0 // indirect; manually updated to v0.6.0, see https://github.com/moby/buildkit/pull/3010#discussion_r942736028
```